docs: listing past versions (#17014)
Lightning-AI/lightning: docs/source-pytorch/past_versions.rst

> Past PyTorch lightning versions

Reviewer: ```suggestion
Past PyTorch Lightning versions
```

===============================

> Pytorch Lightning evolved over time. Here's the full history of PyTorch Lightning versions with links to the respective docs.

Reviewer 2: ```suggestion
PyTorch Lightning evolved over time. Here's the complete history of versions with links to their respective docs.
```

> The Pytorch Lightning evolved over time so to keep track, let us share brief version history list with links to particular docs of the given version.

Reviewer: ```suggestion
Pytorch Lightning evolved over time. Here's the full history of PyTorch Lightning versions with links to the respective docs.
```

.. list-table:: Past versions
   :widths: 10 50 40
   :header-rows: 1

   * - Version
     - Title

>      - Particular Bug-fix versions

Reviewer: ```suggestion
     - Bug-fix versions
```

>    * - `1.9 <https://github.com/Lightning-AI/lightning/releases/tag/1.9.0>`_

Reviewer: we need to list them from last (e.g. 1.9.3) at the top, to first (1.9.0) at the bottom, within each minor

Author: These are in single line
